### PR TITLE
add initial code for HW-7

### PR DESCRIPTION
**Pull request by asamurai**

### CODE REVIEW

**Comment by asamurai** [comment]
Due to sickness, updates can be in 1-2 dates. Please leave your contacts below.

**Comment by asamurai** [comment]
#EVALUATION CRITERIA
- authorization-service is added to the repo, has correct basicAuthorizer lambda and correct serverless.yaml file
- import-service serverless.yaml file has authorizer configuration for the importProductsFile lambda.
- update client application to send Authorization: Basic authorization_token header on import.

#Additional
- Client application should display alerts for the responses in 401 and 403 HTTP statuses.
- FE App: https://d2x1s8t23z38bk.cloudfront.net/
- FE PR: https://github.com/asamurai/nodejs-aws-fe/pull/5

File: authorization-service/.env [hunk 1]
```diff
@@ -0,0 +1 @@
+asamurai=TEST_PASSWORD
```
Comment (BohdanRSchool) [inline authorization-service/.env]: don't push sensitive data to the repo

File: import-service/serverless.yml [hunk 2]
```diff
@@ -57,8 +57,14 @@ functions:
...
               - '*'
+          authorizer:
+            name: tokenAuthorizer
+            arn: ${cf:game-store-authorizarion-service-${self:provider.stage}.basicAuthorizerArn}
+            resultTtlInSeconds: 0
+            identitySource: method.request.header.Authorization
+            type: token
   importFileParser:
-    handler: handler.importFileParser
+    handler: handler.basicAuthorizer
```
Comment (BohdanRSchool) [inline import-service/serverless.yml]: why did you change handler here?
